pkg/combustion/helm.go

```diff
@@ -11,40 +10,50 @@ func ComponentHelmCharts(ctx *image.Context) ([]image.HelmChart, []image.HelmRep
 	}
 
 	const (
-		suseEdgeRepositoryName = "suse-edge"
-		installationNamespace  = "kube-system"
+		metallbRepositoryName = "suse-edge-metallb"
+		metallbNamespace      = "metallb-system"
+
+		endpointCopierOperatorRepositoryName = "suse-edge-endpoint-copier-operator"
```

Comment:
The chances of someone defining a helm repository with this name are incredibly small, but do we want to document that certain names are reserved? Or to make it simpler, maybe just a pattern (i.e. "Don't use the prefix `suse-edge` in any of your user-defined Helm repositories"). I am probably overthinking things, so if you think the chances are low enough to ignore this as a problem, I'm not going to die on this hill.

Reply:
I was thinking about using an underscore prefix but I felt like it's not worth it. Not only the chances of someone using this are astronomical, but the bigger issue here would be someone trying to install the same Helm chart via the definition file.

I believe documenting this would only bring confusion and distraction and the flow would naturally evolve once we flesh out the "first class citizen" deployments.